platform/android/string_android.h

```diff
@@ -58,4 +58,23 @@ static inline String jstring_to_string(jstring source, JNIEnv *env = nullptr) {
 	return result;
 }
 
+/**
+ * Converts Java CharSequence to Godot String.
+ * @param source Source Java CharSequence. If null an empty string is returned.
+ * @param env JNI environment instance. If null obtained by get_jni_env().
+ * @return Godot string instance.
+ */
+static inline String char_sequence_to_string(jobject source, JNIEnv *env = nullptr) {
+	if (source) {
+		if (!env) {
+			env = get_jni_env();
+		}
+
+		jclass cCharSequence = env->GetObjectClass(source);
+		jmethodID toString = env->GetMethodID(cCharSequence, "toString", "()Ljava/lang/String;");
```

Comment:
Camel Case to be consistent with "jni_utils.cpp", e.g.
```
jmethodID getName = env->GetMethodID(cclass, "getName", "()Ljava/lang/String;");
jstring clsName = (jstring)env->CallObjectMethod(cls, getName);
```
But that code is pretty old, maybe that code should be more consistent and not the other way around.